### DISCUSSION TITLE
when we can use winui 2.6 new styles in winui 3?

### DISCUSSION THREAD

**Discussion by ghost1372** [microsoft/WindowsAppSDK#997] [Q&A] [answered]
When will Project Reunion use the new version of WinUI (2.6) with new styles? If I'm not mistaken, Project Reunion is currently using version 2.5.

**Comment by Fabi** [comment, marked as answer]
Anything new on this? We are still stuck with old styles

**Reply by ghost** [reply]
> You can use WinUI 2.6 shipped today to get the latest styles of Windows 11, so UWP apps can look astonishing right now. 
> _**We will integrate these new styles in WinUI 3 (Windows App SDK) by v1.**_

https://twitter.com/marbtweeting/status/1408153511418732544 

**WinUI 3/Windows App SDK v1  -- will integrate WinUI 2.6 styles/controls.**

does it address both of your concerns ?

**Reply by ghost1372** [reply]
> > You can use WinUI 2.6 shipped today to get the latest styles of Windows 11, so UWP apps can look astonishing right now. 
> > _**We will integrate these new styles in WinUI 3 (Windows App SDK) by v1.**_
> 
> https://twitter.com/marbtweeting/status/1408153511418732544 
> 
> **WinUI 3/Windows App SDK v1  -- will integrate WinUI 2.6 styles/controls.**
> 
> does it address both of your concerns ? 
> 
> 

Thank you do you know when we can see preview version? I know that v1 will be released in 2021 Q4

**Reply by ghost** [reply]
it's impossible to say when they will release the preview without having internal knowledge.   

But, according to previous releases' patterns :  

1.0 preview should be out within the first week of Oct. 
1.0 RC within the mid-Oct. 
1.0 GA within mid-Nov.
